crates/cli/src/opts/build/revive.rs

```diff
@@ -10,7 +10,8 @@ pub struct ResolcOpts {
         help = "Enable compiling with resolc",
         long = "resolc-compile",
         visible_alias = "resolc",
-        action = clap::ArgAction::SetTrue,
+        num_args = 0..=1,
+        default_missing_value = "true",
```

Comment:
but wouldn't this always run `resolc` mode?

Reply:
Should be fine, It assume true when --resolc is used.
Now the issue is with positional arguments